lib/init.g

```diff
@@ -927,6 +927,18 @@ BindGlobal ("ShowSystemInformation", function ()
     ShowPackageInformation();
 end );
 
+
+#############################################################################
+##
+## Initialize the IsLIBGAP variable (if not done before). If this variable
+## is false, an interactive session will be started.
+## Otherwise no interactive session is started.
+## This should be done before the system GVAR list is created.
```

Comment:
Change "should" -> "must" Or rather, just drop that line?

Anyway, let's not worry about this and merge as-is; I only mention it in case we need to update this again, say due to a build issue.

Reply:
"Should" is the right word here. It works if you do it afterwards (however that should be done), but it will not be marked as a system gvar.

Reply:
My point is more like this: There are countless variables which "should" or "must" be system vars, and none of them has a comment like this. It feels odd to point this out for just this variable. While I understand that this was a stumbling block during development of this PR, I don't quite see what it adds -- OK, hypothetically it might stop somebody from "moving" it around, but then, why would somebody do that? OTOH, if I just want to understand what it is, then reading this adds cognitive burden.

That said, I am not going to get stuck on that -- I merely point it out.